README.md

```diff
@@ -78,26 +78,65 @@ When puma-dev is installed as a user agent (the default mode), it will log outpu
 
 In the future, puma-dev will provide an integrated console for this log output.
 
-------------
+------
 
 ## Linux Support
 
-Puma-dev supports Linux but requires additional installation to make all the features work. For example, `-install` and `-setup` flags for Linux are not provided, puma-dev root CA is generated but not installed or trusted.
+Puma-dev supports Linux but requires the following additional installation steps to be followed to make all the features work (`-install` and `-setup` flags for Linux are not provided):
+
+### puma-dev root CA
+
+The puma-dev root CA is generated (in `~/.puma-dev-ssl/`), but you will need to install and trust this as a Certificate Authority.
```

Comment:
For Chrome:

1. `Settings`
2. `Advanced` --> `Privacy and security`
3. `Manage certificates`
4. Open tab `Authorities`
5. `Import`
6. Select file `~/.puma-dev-ssl/cert.pem`

Reply:
I decided against putting specific instructions for this step:
* there are multiple ways to do this (adding to the operating system trust store vs. doing so in a specific browser), and the complexities around which browsers use/do not use the operating system store
* multiple browsers (should I add instructions for Firefox, Opera, etc)
* changes that will occur between browser versions.

I will add a bit more details about the options of using the operating system certificate trust store vs. adding directly to the browser which should give enough information to the user to be able to search online for information